Add documentation about the new Serve dashboard features (#33889)
ray-project/ray: doc/source/serve/getting_started.md

## Next Steps

- Dive into the {doc}`key-concepts` to get a deeper understanding of Ray Serve.

> - Look at the details of your Serve application in the Ray Dashboard: {ref}`dash-serve-view`.

Reviewer: ```suggestion
- View details about your Serve application in the Ray Dashboard: {ref}`dash-serve-view`.
```
